src/coretypes/pairScript.sml

```diff
@@ -158,6 +158,13 @@ val PAIR_FST_SND_EQ = store_thm(
 
 val SWAP_def = new_definition ("SWAP_def", ``SWAP a = (SND a, FST a)``)
 
+(* Theorem the SWAP invert itself *)
+Theorem SWAP_SWAP:
```

Comment:
This should be `[simp]`

Reply:
Done.
I'm always hesitant about making really simple lemmas like this simps. Basically, I tend to use them in one theorem or lemma later on, and so I figure the one proof I use them in doesn't justify adding the simplification rule to consideration for every other proof.
That said, if you say it should be a `[simp]`, I am happy to make it as such (since I trust your judgement a LOT more).